parsers/ENTSOE.py

```diff
@@ -926,13 +936,45 @@ def parse_exchange(
         datetime_start = datetime.fromisoformat(
             zulu_to_utc(timeseries.find_all("start")[0].contents[0])
         )
-        # Only use contract_marketagreement.type == A01 (Total to avoid double counting some columns)
```

Comment:
What about this? Are we not risking the double counting if we start ingesting the other forecasts?

Reply:
Both kinds are returned in the same API call so in order to parse it we need to make this dynamic.

Right now it calls the parser function twice, first for the day-ahead values and a second time for the total values to then combine them.

Functionally it should do the same but just use a dynamic config instead of a static value.